manifests/plugin/remote_execution/cockpit.pp

```diff
@@ -48,6 +48,7 @@
   }
 
   foreman_config_entry { 'remote_execution_cockpit_url':
-    value => "${cockpit_path}/=%{host}",
+    value   => "${cockpit_path}/=%{host}",
+    require => Class['foreman::database'],
```

Comment:
Does this ensure that all of the class has executed before this will execute? e.g. db:seed

Reply:
Yes, those resources are contained within the class (if they're managed).

Reply:
Thanks, I just loose track of puppet and how it build it's dependency cycle. Sounds like we could clean this up a bit?

https://github.com/theforeman/puppet-katello/blob/master/manifests/application.pp#L36

Reply:
Yes, that would be better. I've had a look at doing this in the type definition, but since the config part is also used within that, we can't always require the class.